RELEASES.md

```diff
@@ -1,3 +1,131 @@
+
+
+<a id="1.80-Compiler"></a>
+
+Compiler
+--------
+- [Update cc crate to v1.0.97 allowing additional specter mitigations on MSVC targets](https://github.com/rust-lang/rust/pull/124892/)
+- [Allow field reordering on types marked `repr(packed(1))`](https://github.com/rust-lang/rust/pull/125360/)
+- [Detect unused structs even when implementing private traits](https://github.com/rust-lang/rust/pull/122382/)
+- [Add a lint against never type fallback affecting unsafe code](https://github.com/rust-lang/rust/pull/123939/)
+- [Add `--print=check-cfg` to get the expected configs](https://github.com/rust-lang/rust/pull/124320/)
+- [Add `rustfmt` cfg to well known cfgs list](https://github.com/rust-lang/rust/pull/124742/)
+- [Support `Result<T, E>` across FFI when niche optimization can be used](https://github.com/rust-lang/rust/pull/122253/)
+- [Disallow cast with trailing braced macro in let-else](https://github.com/rust-lang/rust/pull/125049/)
+- [Expand `for_loops_over_fallibles` lint to lint on fallibles behind references.](https://github.com/rust-lang/rust/pull/125156/)
+- [self-contained linker: retry linking without `-fuse-ld=lld` on CCs that don't support it](https://github.com/rust-lang/rust/pull/125417/)
+- [Convert `proc_macro_back_compat` lint to an unconditional error.](https://github.com/rust-lang/rust/pull/125596/)
+- [Reject `CVarArgs` in `parse_ty_for_where_clause`](https://github.com/rust-lang/rust/pull/125863/)
+- [Correctly handle type aliases during orphan check](https://github.com/rust-lang/rust/pull/117164/)
+- Improvements to LLDB formatting [#124458](https://github.com/rust-lang/rust/pull/124458) [#124500](https://github.com/rust-lang/rust/pull/124500)
+- [For the wasm32-wasip2 target default to PIC and do not use `-fuse-ld=lld`](https://github.com/rust-lang/rust/pull/124858/)
+- [Adjust 64-bit ARM data layouts for LLVM update](https://github.com/rust-lang/rust/pull/124813/)
+- [Add x86_64-unknown-linux-none target](https://github.com/rust-lang/rust/pull/125023/)
+
+<a id="1.80-Libraries"></a>
+
+Libraries
+---------
+- [Add `size_of` and `size_of_val` and `align_of` and `align_of_val` to the prelude](https://github.com/rust-lang/rust/pull/123168/)
+- [Abort a process when FD ownership is violated](https://github.com/rust-lang/rust/pull/124210/)
+- [io::Write::write_fmt: panic if the formatter fails when the stream does not fail](https://github.com/rust-lang/rust/pull/125012/)
+- [Panic if `PathBuf::set_extension` would add a path separator](https://github.com/rust-lang/rust/pull/125070/)
+- [Add assert_unsafe_precondition to unchecked_{add,sub,neg,mul,shl,shr} methods](https://github.com/rust-lang/rust/pull/121571/)
+- [Update `c_char` on AIX to use the correct type](https://github.com/rust-lang/rust/pull/122986/)
+- [Remove libc from MSVC targets](https://github.com/rust-lang/rust/pull/124050/)
+- [Update libc to 0.2.155](https://github.com/rust-lang/rust/pull/124560/)
```

Comment:
Is libc changes something we want to announce. My only real understanding of libc is that it's some weird C thing and things sometimes don't work when you have it

Reply:
I don't think it affects the "public interface of Rust", but it's an internal change, so maybe in the "Internal Changes" section?

Reply:
I removed the "update libc" since I think that is just an internal thing. I moved "remove libc from msvc targets" to the internal changes since I think that does actually matter. Still really unsure lol.